source/website/src/views/Step3.vue

```diff
@@ -154,10 +154,17 @@ SPDX-License-Identifier: Apache-2.0
             </b-table>
             <b-row>
               <b-col></b-col>
-              <b-col sm="2" align="right" class="row align-items-end">
+              <b-col sm="4" align="right" class="row align-items-end">
+                <b-button v-b-tooltip.hover type="submit" title="Export column schema" variant="outline-secondary" @click="onExport">
+                  Export
+                </b-button> &nbsp;
+                <b-button v-b-tooltip.hover type="submit" title="Import column schema" variant="outline-secondary" @click="onBrowseImports">
+                  Import
+                </b-button>&nbsp;
                 <b-button type="submit" variant="outline-secondary" @click="onReset">
                   Reset
```

Comment:
Small suggestion: Since we added new buttons, I think `Import` and `Export` are now primary actions while `Reset` is a secondary action. In UI design, we should probably have a different button colour for `Reset, since reset is typically considered a secondary action

https://www.oreilly.com/library/view/ux-for-beginners/9781491912676/ch67.html

Reply:
Thank you for the review i have applied the changes you requested.